package.json

```diff
@@ -2,16 +2,16 @@
   "name": "@freecodecamp/freecodecamp",
   "version": "0.0.1",
   "scripts": {
+    "postinstall": "npm run bootstrap",
     "prebootstrap": "npm run ensure-env",
     "bootstrap": "lerna bootstrap && lerna run build --scope @freecodecamp/curriculum",
     "develop": "npm-run-all -s ensure-env start-develop",
     "ensure-env": "cross-env DEBUG=fcc:* node ./tools/scripts/ensure-env.js",
     "lint": "echo 'Warning: TODO - Define Linting.'",
-    "pretest-ci": "npm-run-all -s lint bootstrap",
+    "pretest": "npm-run-all -s lint",
```

Comment:
`npm run lint`

Reply:
Maybe `pretest` replace with `test:lint`?

Reply:
Do you mean this:

```suggestion
    "test:lint": "npm-run-all -s lint",
```

Reply:
Anyways. I will take your advice and update the PR soon so that the final script config looks like so:

```diff
-    "pretest": "npm-run-all -s lint",
+    "pretest": "npm-run-all -s lint:test",
+    "lint: "echo 'Warning: TODO - Define Linting with fixing.'"
+    "test:lint": "echo 'Warning: TODO - Define Linting.'",
```

This should again, keep this open for expansion for the `Linting with fixing.` feature, while achieving the same result, that I intend for.

Would you agree?

Reply:
Thank you for the detailed explanation. I totally agree.